samples/RNWinRTTestApp/README.md

```diff
@@ -15,11 +15,29 @@ The WinRT APIs demonstrated in this sample include:
 
 ## Building and running the sample
 
+1. Open an 'x64 Native Tools Command Prompt for VS 2022' (or whichever version of Visual Studio you have installed)
```

Comment:
does it work in normal command prompt too? I think that's what I've been using

Reply:
Does the `build-npm-package.cmd` step work for you from a normal command prompt? If so, then I guess a normal command prompt is fine, but I've so far been assuming that running msbuild requires it (but perhaps not)

Reply:
The command doesn't work on either normal command prompt or Visual Studio Command Prompt for me, I see 
```
'nuget' is not recognized as an internal or external command,
operable program or batch file.
ERROR: Failed to run nuget restore on rnwinrt.sln
```

Reply:
make sure your VS install includes the nuget command line tools
alternatively i think you can get it from nuget.org, winget, or chocolatey

Reply:
Looks like I had downloaded nuget.exe and manually added it to my path a while ago. Looking around, it seems as though `msbuild rnwinrt\rnwinrt.sln -t:Restore -p:RestorePackagesConfig=true` yields the same behavior

Reply:
yeah, I didn't have nuget.exe in my Environment Variables path. After I added it, I still needed to run on VS Command Prompt to recognize msbuild. I think replacing with the `msbuild` command would be better

Reply:
The script should now be updated

Reply:
Thanks :)  It works for me!